exporter/awsxrayexporter/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter
 
-go 1.12
+go 1.14
```

Comment:
hum every time we update we have to change those, I'm thinking that a better practice for this repo will be to remove the version from the module.

Reply:
we can do it separately to not confuse things

Reply:
+1 to do it separately. My understanding though is if you don't add it, `go mod tidy` will.

Reply:
I'm not entirely sure but I think we don't strictly need to keep updating this. This sort of refers to the minimum Go version that can build this project and AFAIK, the compiler does not enforce anything using this directive. AFAIK,  it is only used to provide extra information/hints to users when a build fails.